src/SWP/Bundle/FacebookInstantArticlesBundle/composer.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "swp/facebook-instant-articles-bundle",
+  "type": "symfony-bundle",
+  "description": "This bundle provides Facebook Instant Articles support for Symfony applications.",
+  "keywords": ["facebook", "instant-articles", "superdesk", "webpublisher", "publisher", "fbia"],
+  "license":"AGPL-3.0",
+  "minimum-stability": "dev",
+  "authors": [
+    {
+      "name": "Paweł Mikołajczuk",
+      "email": "pawel.mikolajczuk@sourcefabric.org"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "path",
+      "url": "../*"
+    },
+    {
+      "type": "path",
+      "url": "../../Component/*"
```

Comment:
It's for telling composer to use components from main repo (if called from repo in tests) instead downloading last released version from packagist. It's needed when you test changes in your bundle and they rely on changes in one of our components (no merged yet).